docs/notification-routing/disable-oomkill-notifications.rst

```diff
@@ -0,0 +1,11 @@
+Disable "OOMKill" Notifications
+===================================
+
+Configure Robusta to not send OOMKill notifications by disabling the built-in OOMKill playbook. 
+
+.. code-block:: yaml
+
+    disabledPlaybooks:
+    - PodOOMKill
+
+However you can still create your custom OOMKill notification using the ``kube_pod_container_status_last_terminated_reason{reason="OOMKilled"}`` Prometheus metric.
```

Comment:
I think you can remove this part. The whole point is disabling it, and this just confuses the user without an explanation of when to do it, etc.

Can you also explain that any other builtin notification can be disabled by name that way, and link to the relevant part of the helm values in github where users can see other builtin playbooks.